.github/test-spec.yml

```diff
@@ -205,7 +205,30 @@
   - "drivers/mpsl/**/*"
 
 "CI-ble-samples-test":
-  - "samples/bluetooth/**/*"
```

Comment:
Can we use a blacklist for dragoon specific samples that run in dragoon CI, so when more  (non-dragoon) bluetooth samples are added to automation, the CI can be triggered by default.
  - any:
      - "samples/bluetooth/**/*"
      - "!samples/bluetooth/scanning_while_connecting/*"
      - "!samples/bluetooth/subrating/*"
      -  ...

Reply:
yes, i think blacklist can be used

Reply:
@rafpelc , @guwa I don't have a strong preference here, but I see that all CI plans use accept list instead of deny lists. Shouldn't we stick to that convention?